pkg/scheduler/framework/plugins/enqueue/scheduling_gates.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package defaultenqueue
+package enqueue
```

Comment:
I would name the folder and pkg `schedulinggates` too, just like other features. Remember we could have other enqueue plugins, just like `preFilter` etc.